metalinter.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Linters": {
+	      "errcheck": "errcheck -abspath -ignore 'Close|Write|Serve' {path}:PATH:LINE:COL:MESSAGE",
```

Comment:
I think this works for now. Once we enable errcheck and we start seeing os.Remove related errors, we can look for a fix.